### PR TITLE
Board page

### PR DESCRIPTION
**Pull request by kiminlee2000225**
Might fix the Linkedin / email icon in the future to be easily viewed with a black background.

### CODE REVIEW

File: src/components/Board.tsx [hunk 1]
```diff
@@ -14,20 +18,94 @@ const useStyles = makeStyles({
...
+        <Helmet>
+          <title>E-BOARD</title>
+          <meta
+            name="description"
+            content="The Code4Community E-Boards page."
+          />
+        </Helmet>
+        <Hero
+          title="E-Board"
+          subtitle="Meet the people of Code 4 Comunity"
```
Comment (jackblanc) [inline src/components/Board.tsx]: "Meet the people of Code4Community"

File: src/components/Board.tsx [hunk 2]
```diff
@@ -14,20 +18,94 @@ const useStyles = makeStyles({
...
-        <Paper className={classes.example} />
-      </Box>
-    </Container>
+    <div>
+      <Container maxWidth="md" className={classes.hero}>
+        <Helmet>
+          <title>E-BOARD</title>
+          <meta
+            name="description"
+            content="The Code4Community E-Boards page."
```
Comment (jackblanc) [inline src/components/Board.tsx]: make this the same as subtitle below

File: src/components/Board.tsx [hunk 3]
```diff
@@ -14,20 +18,94 @@ const useStyles = makeStyles({
...
+          />
+        </Helmet>
+        <Hero
+          title="E-Board"
```
Comment (jackblanc) [inline src/components/Board.tsx]: "People" or "Team" or something, not eboard specific

File: src/components/Board.tsx [hunk 4]
```diff
@@ -14,20 +18,94 @@ const useStyles = makeStyles({
...
-        <meta name="description" content="Write a description for SEO here" />
-      </Helmet>
-      <Box my={4}>
-        <Paper className={classes.example} />
-      </Box>
-    </Container>
+    <div>
+      <Container maxWidth="md" className={classes.hero}>
+        <Helmet>
+          <title>E-BOARD</title>
```
Comment (jackblanc) [inline src/components/Board.tsx]: Make this reflect the Hero title

File: src/components/Member.tsx [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    name: {
+      fontWeight: 550,
+      paddingBottom: 4,
+      fontSize: 11.5,
+    },
+  },
+}));
+
+interface MemberProps {
+  name: string;
```
Comment (jackblanc) [inline src/components/Member.tsx]: `readonly name: string;`

File: src/components/Member.tsx [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+          <a href={linkedin} rel="noopener noreferrer" target="_blank">
+            <img
+              className={classes.linkedin}
+              src="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/icons/linkedin_icon.jpg"
+            />
+          </a>
+          <a href={'mailto:' + email} rel="noopener noreferrer" target="_blank">
+            <img
+              className={classes.mail}
+              src="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/icons/mail_icon.jpg"
```
Comment (jackblanc) [inline src/components/Member.tsx]: we may prefer an SVG for this. thoughts @rymaju

**Review by jackblanc — COMMENTED** [review]
Left a few comments, this looks awesome 👍  🎉

File: src/components/Member.tsx (hunk 6 above)
Comment (rymaju) [reply src/components/Member.tsx]: Yes! + SVGs can be whatever color we want, are lighter, and have transparency. makes a lot of sense

File: public/sitemap.xml [hunk 7]
```diff
@@ -17,6 +17,9 @@
   <url>
     <loc>https://c4cneu.com/jumpstart</loc>
   </url>
+  <url>
+    <loc>https://c4cneu.com/board</loc>
```
Comment (rymaju) [inline public/sitemap.xml]: change to https://c4cneu.com/people

File: src/components/Board.tsx [hunk 8]
```diff
@@ -1,33 +1,120 @@
...
-      </Helmet>
-      <Box my={4}>
-        <Paper className={classes.example} />
-      </Box>
-    </Container>
+    <div>
+      <Container maxWidth="md" className={classes.hero}>
+        <Helmet>
+          <title>People</title>
+          <meta name="People" content="Meet the people of Code4Comunity" />
```
Comment (rymaju) [inline src/components/Board.tsx]: misspelled also add a period here? ie "Meet the people of Code4Community."

**Review by rymaju — COMMENTED** [review]
Two little typos away from being merged!

File: src/components/Board.tsx [hunk 9]
```diff
@@ -39,11 +39,11 @@ const Template: React.FC = () => {
       <Container maxWidth="md" className={classes.hero}>
         <Helmet>
           <title>People</title>
-          <meta name="People" content="Meet the people of Code4Comunity" />
+          <meta name="People" content="Meet the people of Code4Comunity." />
```
Comment (rymaju) [inline src/components/Board.tsx]: Code4Comunity is still misspelled. its Code4Community

File: src/components/Board.tsx [hunk 10]
```diff
@@ -39,11 +39,11 @@ const Template: React.FC = () => {
...
         </Helmet>
         <Hero
           title="People"
-          subtitle="Meet the people of Code4Comunity"
+          subtitle="Meet the people of Code4Comunity."
```
Comment (rymaju) [inline src/components/Board.tsx]: Same here

**Review by rymaju — APPROVED** [review]
LGTM 🎉 😄

**Pull request merged**